services/credits-service/src/services/tigerbeetle_client.py

```
@@ -0,0 +1,248 @@
+        except Exception as e:
+            logger.error(f"Failed to connect to TigerBeetle: {e}")
+            raise DatabaseConnectionException(f"Failed to connect to TigerBeetle: {e}") from e
+
+    async def disconnect(self) -> None:
+        """Disconnect from TigerBeetle"""
+        if self._client:
+            try:
+                # close() returns None, no need to await
+                self._client.close()
+                logger.info("Disconnected from TigerBeetle")
+            except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)